Makefile

```
@@ -1,5 +1,6 @@
 GIT_COMMIT?="$(shell git rev-parse HEAD | head -c 7)"
-NAME_POSTFIX?="$(shell docker ps -a | wc -l | xargs)"
+CONTAINER_ENGINE?=$(shell podman version >/dev/null 2>&1 && echo podman || echo docker)
```

```suggestion
CONTAINER_ENGINE?="docker"
```
Let's have it as an outside parameter for cases in which docker is not available (it  is available on Linux), don't think we want an automatic (and default) behaviour which can change the engine we build with.

---

The whole point was to prefer podman over docker, if available.
If it doesn't make sense, I can drop this.
The reason I needed this was that Docker is disabled on my Fedora as I'm using podman (without root privs., etc.).

---

I think that supporting podman is good. I don't think it should be preferred.
So adding a parameter to use it so people without docker would be able to build is good.